tools/config/go-templates/api.mustache

```diff
@@ -48,7 +48,7 @@ type {{classname}} interface {
 	{{nickname}}WithParams(ctx context.Context, args *{{operationId}}ApiParams) {{operationId}}ApiRequest
 
 	// Interface only available internally
```

Comment:
is this still true?

Reply:
Yes. Maybe I should change message to make method available for mocking

```suggestion
	// Method available only for mocking purposes
```

Reply:
why do we need this method on the interface? can we keep it private and remove from interface? is it needed for mocking?

Reply:
Answered inline: https://github.com/mongodb/atlas-sdk-go/pull/262/files#r1459286558